features/kinderen.feature

```diff
@@ -1,71 +1,75 @@
-    En heeft dit kind naam.voornamen=Franklin
-    En heeft dit kind naam.voorvoegsel=null
-    En heeft dit kind naam.geslachtsnaam=Groenen
-    En heeft dit kind naam.adellijkeTitel_predikaat.omschrijvingAdellijkeTitel_predikaat=null
-    En heeft dit kind geboorte.datum.datum=1983-05-26
-    En heeft dit gevonden kind de ingeschrevenpersonen link met /ingeschrevenpersonen/999999047
+    Scenario: naamswijziging kind
+      Gegeven de persoon met burgerservicenummer 999996381 heeft de volgende kinderen in de registratie
+        | Categorie | Voornamen (02.10) |
+        | 9         | Vica              |
+        | 59        | Celeke Lodivica   |
+        | 9         | Bella             |
+      Als de ingeschreven persoon met burgerservicenummer 999996381 wordt geraadpleegd met fields=kinderen
+      Dan bevat het antwoord 2 kinderen
+      En bevat het antwoord het kind met naam.voornamen "Vica"
+      En bevat het antwoord het kind met naam.voornamen "Bella"
+      En bevat het antwoord GEEN kind met naam.voornamen "Celeke Lodivica"
+  
 
-  Scenario: het kind is geen ingeschreven persoon
-    Gegeven de te raadplegen persoon heeft een kind die zelf geen ingeschreven persoon is
-    En het kind van de ingeschreven persoon heeft volgens categorie 09/59 naam Chantal Groenen, geboren in Istanbul en geboortedatum 2 januari 2002
-    Als de kinderen worden geraadpleegd van de ingeschreven persoon met burgerservicenummer 999999291
-    Dan wordt het kind gevonden met naam.voornamen=Chantal
-    En heeft dit kind burgerservicenummer=null
-    En heeft dit kind naam.voorvoegsel=null
-    En heeft dit kind naam.geslachtsnaam=Groenen
-    En heeft dit kind naam.adellijkeTitel_predikaat.omschrijvingAdellijkeTitel_predikaat=null
-    En heeft dit kind geboorte.datum.datum=2002-01-02
-    En heeft dit kind geboorte.datum.dag=02
-    En heeft dit kind geboorte.datum.maand=01
-    En heeft dit kind geboorte.plaats=Istanbul
-    En heeft dit kind geboorte.land.landnaam=Turkije
-    En heeft dit gevonden kind een lege link ingeschrevenpersonen
+  Rule: Een kind wordt alleen teruggegeven als minimaal één gegeven in de identificatienummers (groep 01), naam (groep 02) of geboorte (groep 03) van het kind een waarde heeft.
+    # wanneer in een categorie kind alleen gegevens zijn opgenomen in groep 81 of 82, 85 en 86, wordt dit kind niet opgenomen in het antwoord
+    # wanneer een gegeven een onbekendwaarde heeft, zoals . (punt) bij geslachtsnaam en 00000000 bij geboortedatum, wordt die in deze regel beschouwd als leeg (geen waarde).
 
-  Scenario: de ingeschreven persoon twee kinderen
-    Gegeven de te raadplegen persoon heeft meerdere (twee) kinderen (Lisa en Daan)
-    Als de kinderen worden geraadpleegd van de ingeschreven persoon met burgerservicenummer 999999047
-    Dan is het aantal gevonden kinderen 2
-    En wordt het kind gevonden met naam.voornamen=Lisa
-    En wordt het kind gevonden met naam.voornamen=Daan
+    Scenario: kind volledig onbekend
```

Comment:
Wat betekent dit functioneel? Is dit een data fout of kan dit echt voorkomen? Als het echt kan voorkomen, mag een consumer van de API zomaar concluderen dat deze persoon geen kinderen heeft?

Reply:
@CathyDingemanse  zoekt dat uit, zie https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/pull/848#discussion_r78394938